.github/workflows/lint.yml

```diff
@@ -0,0 +1,57 @@
+name: lint
+
+on:
+  push:
+    branches:
+    - develop
+    - stable
+  pull_request:
+    branches:
+    - develop
+    - stable
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
v4?

Reply:
Right, this is only updated on develop so far. I don't want to manually update all of these, but we could configure dependabot to also send updates for the stable branch, if you want.

Reply:
I mentioned it because of the new lines.
Yes adding the bot for stable is a good thing. A stable release seems is renewed less frequently than the bot architecture.

Reply:
I filed #2339.